### ISSUE TITLE
Issue name Issues while launching ur_robot_driver

### ISSUE THREAD

**Issue by neerja123-gupta** [UniversalRobots/Universal_Robots_ROS2_Driver#959]
### Affected ROS2 Driver version(s)

ur_robot_driver

### Used ROS distribution.

Humble

### Which combination of platform is the ROS driver running on.

Ubuntu Linux with standard kernel

### How is the UR ROS2 Driver installed.

From binary packets

### Which robot platform is the driver connected to.

UR E-series robot

### Robot SW / URSim version(s)

UR10e

### How is the ROS driver used.

Through the robot teach pendant using External Control URCap

### Issue details

Hi All, I am using ROS2 humble on ubuntu 22.04. I cloned this repo in my local system under ros_ws, it built fine there was no problem in the build, and also I installed ur_robot_driver from binary package using sudo apt-get install ros-humble-ur. However, on launching the driver ros2 launch ur_robot_driver ur_control.launch.py ur_type:=ur10e robot_ip:=robot_ip I face this issue Waiting for '/controller_manager' node to exist. Any help would be highly appreciated. Thanks

### Relevant log output

```shell
pawner-11] [INFO] [1712245658.539314013] [spawner_force_torque_sensor_broadcaster]: Waiting for '/controller_manager' node to exist
[spawner-12] [INFO] [1712245658.568346962] [spawner_forward_position_controller]: Waiting for '/controller_manager' node to exist
[spawner-9] [INFO] [1712245658.581252250] [spawner_io_and_status_controller]: Waiting for '/controller_manager' node to exist
[spawner-10] [INFO] [1712245658.590848760] [spawner_speed_scaling_state_broadcaster]: Waiting for '/controller_manager' node to exist
[spawner-8] [INFO] [1712245658.601270697] [spawner_joint_state_broadcaster]: Waiting for '/controller_manager' node to exist
[spawner-7] [INFO] [1712245658.612340232] [spawner_scaled_joint_trajectory_controller]: Waiting for '/controller_manager' node to exist
[spawner-11] [INFO] [1712245660.560955491] [spawner_force_torque_sensor_broadcaster]: Waiting for '/controller_manager' node to exist
[spawner-12] [INFO] [1712245660.587354638] [spawner_forward_position_controller]: Waiting for '/controller_manager' node to exist
[spawner-9] [INFO] [1712245660.602333906] [spawner_io_and_status_controller]: Waiting for '/controller_manager' node to exist
[spawner-10] [INFO] [1712245660.611685941] [spawner_speed_scaling_state_broadcaster]: Waiting for '/controller_manager' node to exist
[spawner-8] [INFO] [1712245660.623131296] [spawner_joint_state_broadcaster]: Waiting for '/controller_manager' node to exist
[spawner-7] [INFO] [1712245660.633942299] [spawner_scaled_joint_trajectory_controller]: Waiting for '/controller_manager' node to exist
[spawner-11] [INFO] [1712245662.582232820] [spawner_force_torque_sensor_broadcaster]: Waiting for '/controller_manager' nod
```

### Accept Public visibility

- [X] I agree to make this context public

**Comment by RobertWilbrandt** [comment]
As i understood this you built the driver from sources and installed everything else from binary packages. Please follow our [Getting started](https://github.com/UniversalRobots/Universal_Robots_ROS2_Driver/tree/humble?tab=readme-ov-file#getting-started) section and either use everything from binary packages or build all relevant direct dependencies from source as well (as described there). If you install the driver from binary packages you don't need to clone this repository at all.

It seems like there is some problem with the driver starting. Could you please provide the full log output so we can see what the underlying cause to this could be?

**Comment by neerja123-gupta** [comment]
Hi @RobertWilbrandt , thanks for your reply. Yes, so I installed ros2 humble using Debian/binary packages and then I cloned this repository humble branch and build it from source. Yes, so this issue "Waiting for '/controller_manager' node to exist" is random, so whenever we change the netmask of the robot the issue disappears. So sometimes it works on netmask 255.255.0.0 and sometimes on 255.255.255.0. Not able to understand the exact cause of this error.
